3rdpartylibrary/google-maps-android-cmgn/src/main/java/com/google/maps/android/kml/KmlFeatureParser.java

```diff
@@ -86,9 +87,11 @@ static KmlPlacemark createPlacemark(XmlPullParser parser)
     /**
      * Creates a new GroundOverlay object (created if a GroundOverlay tag is read by the
      * XmlPullParser) and assigns specific elements read from the parser to the GroundOverlay
+     * @param documentBase Path leading to kml file directory to generate full path file url for map service.
+     *                     For example if the file was in C:/Documents/MyKMZ.kmz/test.kml this value would be C:/Documents/MyKMZ.kmz
      */
     /* package */
-    static KmlGroundOverlay createGroundOverlay(XmlPullParser parser)
+    static KmlGroundOverlay createGroundOverlay(XmlPullParser parser, String documentBase)
```

Comment:
I forgot to mention this before but please add the word final before all variables (for this part it should be in front of the parameters)